src/components/payment/buy-form.tsx

```diff
@@ -0,0 +1,163 @@
+import { useState } from 'react';
+import type { SubmitErrorHandler } from 'react-hook-form';
+import { FormProvider } from 'react-hook-form';
+import { z } from 'zod';
+
+import { withFormHOC } from '@/core/hoc/hoc-form';
+import { FormInputs } from '@/core/hoc/multiple-input';
+import { Button, Checkbox, ScrollView, Text, View } from '@/ui';
+
+import { DatePicker, schema as dateSchema } from '../date-picker';
+import { HeaderLogo } from '../header-logo';
+
+export const _adressSchema = z.object({
+  city: z.string({
+    required_error: 'City required',
+  }),
+  country: z.string({
+    required_error: 'Country required',
+  }),
+  adress1: z.string({
```

Comment:
typo, should be address, check the rest of the code as it's a misspelled throughout the code